requirements.txt

```diff
@@ -1,6 +1,5 @@
 # production
 Django>=1.4
-django-timedeltafield==0.7.2
```

Comment:
Why did you replace this lib with a custom one, made from scratch one? Just wonder why :)

Reply:
the old library imported things from django that are not available anymore and it's not maintained anymore.
Since the only use of the library was the parse function i just copied the code and removed al references to six and incompatible code.

Reply:
Nice!